Dockerfile.ci

```
@@ -1,12 +1,29 @@
 FROM dependabot/dependabot-core
-ARG CODE_DIR=/home/dependabot/dependabot-core
+
+USER root
+
+ARG DEPENDABOT_HOME=/home/dependabot
+ARG USERNAME=dependabot
```

```suggestion
ARG USERNAME=dependabot
ARG DEPENDABOT_HOME=/home/${USERNAME}
```

Minor, matches `Dockerfile.development`